UnityProjectDP/Assets/Resources/Animations/Observer-VaccinationCase copy.json

```diff
@@ -1,150 +0,0 @@
-{
```

Comment:
preco sme zmazali priklady animacii?

Reply:
Myslel som ze copy je kopia, but i stand corrected je to späť